### PR TITLE
fix(profile page):Validation of newly added field in profile page

### PR DESCRIPTION
**Pull request by gayatri200**
### Functionality:
Validation of newly added field in profile page

### Solution:
Adding validation service in profile page 

### Risk level:
- [ ] high 
- [x] medium
- [ ] low

### How to test:
In Education,work experience,projects and skill ,without adding info if we click on submit it is showing validation error

### CODE REVIEW

File: src/app/body/profile/edit-education/edit-education.component.ts [hunk 1]
```diff
@@ -39,7 +41,51 @@ export class EditEducationComponent implements OnInit {
...
+  async updateEducation() {
+    let labels = ['instituteName', 'degree', 'startDate', 'endDate'];
+    let values = [this.instituteName, this.degree, this.startDate, this.endDate];
+    let data = [{ label: "instituteName", value: this.instituteName },
+    { label: "degree", value: this.degree },
+    { label: "startDate", value: this.startDate },
+    { label: "endDate", value: this.endDate }];
+   
+    var condition = await (this.validationService.checkValidity(this.componentName, data)).then(res => {
+      console.log("edit");
```
Comment (Pendyala-thanvi) [inline src/app/body/profile/edit-education/edit-education.component.ts:53]: Remove these consoles.

File: src/app/body/profile/edit-education/edit-education.component.ts [hunk 2]
```diff
@@ -39,7 +41,51 @@ export class EditEducationComponent implements OnInit {
...
+    let labels = ['instituteName', 'degree', 'startDate', 'endDate'];
+    let values = [this.instituteName, this.degree, this.startDate, this.endDate];
+    let data = [{ label: "instituteName", value: this.instituteName },
+    { label: "degree", value: this.degree },
+    { label: "startDate", value: this.startDate },
+    { label: "endDate", value: this.endDate }];
+    
+    var condition = await (this.validationService.checkValidity(this.componentName, data)).then(res => {
+      console.log("edit");
+      console.log(this.componentName);
```
Comment (Pendyala-thanvi) [inline src/app/body/profile/edit-education/edit-education.component.ts:76]: Same here

File: src/app/body/profile/edit-education/edit-education.component.ts (hunk 1 above)
Comment (gayatri200) [reply src/app/body/profile/edit-education/edit-education.component.ts:53]: done

File: src/app/body/profile/edit-education/edit-education.component.ts (hunk 2 above)
Comment (gayatri200) [reply src/app/body/profile/edit-education/edit-education.component.ts:76]: done

**Review by Pendyala-thanvi — APPROVED** [review]
Approved

**Review by viv9k — APPROVED** [review]
Approved

**Pull request merged**